RawSpeed/LJpegDecompressor.cpp

```diff
@@ -463,27 +429,25 @@ void LJpegDecompressor::createHuffmanTable(HuffmanTable *htbl) {
  ************************************/
 
 void LJpegDecompressor::createBigTable(HuffmanTable *htbl) {
-  const uint32 bits = 14;      // HuffDecode functions must be changed, if this is modified.
+  const uint32 bits = HuffmanTable::TableBitDepth;      // HuffDecode functions must be changed, if this is modified.
   const uint32 size = 1 << bits;
-  int rv = 0;
-  int temp;
-  uint32 l;
 
   if (!htbl->bigTable)
     htbl->bigTable = (int*)_aligned_malloc(size * sizeof(int), 16);
```

Comment:
I have a couple of issues with this suggestion:
1. it has nothing to do with the code changes in this PR (I did not touch this line)
1. it has nothing to do with the subject of the respective commit
1. the order of the operands in a int multiplication of two numbers is irrelevant. It produces the same exact assembler code: https://godbolt.org/g/5YCvtk
1. this particular multiplication could never overflow anyway, because both operands are compile time constants

Lets discuss/work on the 'hardening' of the code with respect to potential overflows in a separate effort.